.gitignore

```diff
@@ -5,4 +5,3 @@ dist
 generated
 e2e/screens
 node_modules
-package-lock.json
```

Comment:
I had this file previously versioned, but is not recommended for libraries as someone told me, something like this: https://dev.to/gajus/stop-using-package-lock-json-or-yarn-lock-3ddi

As mentioned on that blog post, if you publish any package to NPM it shall not contain the package-lock.json anymore! So, our consumers had no benefit of having this file versioned.

Also, by experience, I noticed you can get flooded of security reports of dependencies that are not actually used!

I am still unconvinced of this, but if it can work for our collaborators I think it'll be fine, what do you think?

Reply:
okay, I used to see that file versioned in other libraries - that being said I'm fine without, let's keep the discussion open and revisit if it need to

I'll update the PR without it